src/components/universal/core/Image/Image.tsx

```diff
@@ -0,0 +1,18 @@
+import { panda } from "generated/panda/jsx";
```

Comment:
Remove this file. Must have been a merge conflict that was not resolved correctly during the `git cherry-pick`s... that is my bad!

Reply:
Remove the `Image` directory actually. Anything related to `Image` should not be in this PR and that is definitely my bad!